### PR TITLE
Issue #2418 Not printing extensions if none exist

### PR DESCRIPTION
**Pull request by aarb1**
Issue #2418 Not printing extensions if none exist

### CODE REVIEW

**Comment by wraschke** [comment]
From Erik:

> Sorry for the confusion on the checklist, this is 1 of 6 PRs that were affected by the rollout of checklists for openliberty. It's not quite ready yet and being turned off, so for this PR please ignore the 'yellow' status check below, but do be on the lookout for future changes to how PRs are created and managed.

File: dev/com.ibm.ws.logging/src/com/ibm/ws/logging/source/LogSource.java [hunk 1]
```diff
@@ -155,18 +155,18 @@ public GenericData parse(RoutedMessage routedMessage) {
         genData.addPair("ibm_methodName", logRecord.getSourceMethodName());
         genData.addPair("ibm_className", logRecord.getSourceClassName());
 
-        KeyValuePairList extensions = new KeyValuePairList();
         Map<String, String> extMap = null;
```
Comment (donbourne) [inline dev/com.ibm.ws.logging/src/com/ibm/ws/logging/source/LogSource.java]: minor comment - should move declaration of extMap down to line below where it is set.  keeps code neater as then you don't need to think whether extMap is used in code later when you are reading the code.

**Comment by aarb1** [comment]
#build

**Comment by aarb1** [comment]
#build

**Pull request merged**